frontend/src/components/AppHeader.js

```diff
@@ -1,25 +1,32 @@
 import React from "react";
 import {Link} from "react-router-dom";
+import {BiArrowBack} from "react-icons/all";
+import styled from "styled-components/macro";
 
 
 export default function AppHeader() {
 
     return (
-        <>
+        <Wrapper>
             <Link to="/">
-                <img
-                    src="/images/C.svg"
-                    alt="logo"
-                    style={{
-                        position: "absolute",
-                        top: -148,
-                        height: "100%",
-                        width: "100%",
-                        strokeOpacity: 30,
-                    }}
-                />
+                <BiArrowBack className="arrow" size={30}/>
             </Link>
-        </>
+        </Wrapper>
     )
 
-}
\ No newline at end of file
+}
+
+
+
+const Wrapper = styled.div`
+  
+  margin-right: 320px;
```

Comment:
Why the big margin to the right, I would expect a header do be full width?

Reply:
Had to adjust the margin that big because the header only consist of the icon 😅  my hamburger menu isn't working yet :(